### ISSUE TITLE
Recommendation: Indicate to user which micronutrient is the recommendation suppose to accommodate for

### ISSUE THREAD

**Issue by ericr491** [STRIDES-Codes/Uncovering-the-Nutritional-Landscape-of-Food#3]

**Comment by Kyle-Pu** [comment]
`lackingNutrient = min(data2['Total'], key=data2['Total'].get)` in [app.py](https://github.com/STRIDES-Codes/Uncovering-the-Nutritional-Landscape-of-Food/blob/main/app.py) holds the current nutrient recommendations are showing for.

**Comment by ericr491** [comment]
Yep, and adding on I think it would be nice if the name of the nutrient is displayed to the user, especially apparent if the user does not select all the vitamins as the 'lowest' could be a vitamin that does not show in the graph.

**Comment by Kyle-Pu** [comment]
Yeah great point
